src/applications/static-pages/situation-updates-banner/createSituationUpdatesBanner.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { useFeatureToggle } from '~/platform/utilities/feature-toggles';
+import widgetTypes from '../widgetTypes';
+import SituationUpdateBanner from './situationUpdateBanner';
+
+export const BannerContainer = () => {
+  const {
+    TOGGLE_NAMES,
+    useToggleValue,
+    useToggleLoadingValue,
+  } = useFeatureToggle();
+
+  const alternativeBannersEnabled = useToggleValue(
+    TOGGLE_NAMES.bannerUseAlternativeBanners,
+  );
+  const isLoadingFeatureFlags = useToggleLoadingValue();
+
+  if (isLoadingFeatureFlags || !alternativeBannersEnabled) {
+    return null;
+  }
+
+  const defaultProps = {
```

Comment:
The banner component uses these props: `id, alertType, headline, showClose, content`. Where are all the other props in this object used?

Reply:
These end up being used by the Banner component itself `src/applications/static-pages/situation-updates-banner/situationUpdateBanner.jsx`

Reply:
They aren't being used in this static V1 version but are all pieces that are intended to be fetched and stored in the DB. I expect we may want to remove a few of these before this is 100% done, but I'm open to removing the extras for now if that feels too crufty

Reply:
Thanks for clearing this up! All good.